deeds-dapp-webapp/src/main/webapp/vue-app/snapshot/components/assets/DeedAssets.vue

```diff
@@ -17,19 +17,62 @@
  Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 -->
 <template>
-  <v-list dense v-if="ownedNfts && ownedNfts.length">
+  <v-list dense>
     <v-list-item>
       <h4>{{ $t('yourDeeds') }}</h4>
     </v-list-item>
-    <v-list-item
-      v-for="nft in nftsByCardType"
-      :key="nft.id"
-      class="ps-8">
-      <v-list-item-content>{{ nft.cardName }}</v-list-item-content>
-      <v-list-item-content class="align-end">{{ nft.cityName }}</v-list-item-content>
-      <v-list-item-content class="align-end">{{ nft.count }}</v-list-item-content>
-      <v-list-item-content class="align-end" />
-    </v-list-item>
+    <v-skeleton-loader
+      v-if="!ownedNfts"
+      type="image"
+      class="mx-auto ma-2"
+      max-width="90%" />
+    <div v-else>
+      <template
+        v-if="ownedNfts && ownedNfts.length">
```

Comment:
`ownedNfts` will never be null here since you have a `v-if` on it above. BTW, that's why it's better to use `computed` to make the code more clear and comprehensible
```javascript
computed: {
  nftsLoading() {
    return !!this.ownedNfts;
  },
  ownsNFTs() {
    return !this.nftsLoading && this.ownedNfts?.length;
  },
}
```